tests/karma/karma.conf.local-launchers.js

```diff
@@ -2,9 +2,9 @@ export default function () {
 
   return {
     browsers: [
-      //'Chrome',
+      'Chrome',
       'Firefox',
-      //'HeadlessChrome'
+      'HeadlessChrome'
```

Comment:
What is the benefit of running headless chrome as well as standard chrome?

Reply:
Part of previous configuration. I haven't chosen to remove it.